fns-hl7-pipeline/fn-lake-segs-transfomer/src/test/resources/lakeSegsModelSample.json

```diff
@@ -0,0 +1,41 @@
+[
```

Comment:
I'm reviewing your PR, and I see the following model on tests (assuming that's what you're generating)

[
    "MSH|^~\\&|SendAppName",
    "PID|1||HEP_B_TC02"
  ],
  [
    "MSH|^~\\&|SendAppName",
    "OBR|1|\"\"|HEP_B_TC02",
    "OBX|1.1|CWE|78746-5^Country of Birth"
  ],
We are missing some info there - 

1 - I would like to have the line number associated with each Segment

2 - I need an entry for MSH alone (not sure if that was somewhere else)

3 - I'd like to separate the Segment in question from the array of the B-line to the root....



ON the ticket we had this schema:

{
   "ref": "{SEG}-99",
   "segment": "...",
   "line_number": 99,
   "parentSegmentRef": "" 
}
​we can punt on the ref attribute for now, but would like segment  and line number to be there and parentSegmentRef can be the arrays you have - minus the segment itself. (so for MSH parentSegmentRef is null)


and since we're not using references, we can call it parentSegments.